registration-processor/registration-processor-landing-zone-Utility/Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM openjdk:11
```

Comment:
Since you are reading packets from LANDING_ZONE location, you have to do a volume mapping of LANDING_ZONE in dockerfile. Check the stage-group-1 dockerfile for same.

Reply:
I have added the volume